projects/1.18/assets/miners-lung/minerslung/lang/zh_cn.json

```
@@ -0,0 +1,32 @@
+{
+  "item.minerslung.respirator": "呼吸面罩",
+  "item.minerslung.air_bladder": "气囊",
+  "item.minerslung.soulfire_bottle": "灵魂火瓶",
+  "block.minerslung.safety_lantern": "安全灯",
+  "advancement.minerslung:root": "矿工之肺",
+  "advancement.minerslung:root.desc": "主世界的深处和其他维度的空气不总是可以呼吸的",
+  "advancement.minerslung:air_bladder": "来来去去",
+  "advancement.minerslung:air_bladder.desc": "在旅途中使用气囊补充空气",
+  "advancement.minerslung:soul": "深蓝",
+  "advancement.minerslung:soul.desc": "呼吸灵魂火释放的生命力来保持你的氧气值（但并不会增加它！）",
+  "advancement.minerslung:soulfire_bottle": "以防万一",
+  "advancement.minerslung:soulfire_bottle.desc": "用囚禁在灵魂火瓶中的灵魂恢复你的肺",
+  "advancement.minerslung:protection_from_yellow": "我得深深地吸口气······",
+  "advancement.minerslung:protection_from_yellow.desc": "用类似呼吸面罩之类的东西保护你免受窒息",
+  "advancement.minerslung:protection_from_red": "……然后屏住呼吸",
+  "advancement.minerslung:protection_from_red.desc": "在没有空气的地方自由地呼吸。",
+  "advancement.minerslung:lantern": "我们的第一优先事项",
+  "advancement.minerslung:lantern.desc": "使用安全灯检查你附近的空气质量",
+  "advancement.minerslung:disco_lantern": "红灯！绿灯！",
+  "advancement.minerslung:disco_lantern.desc": "使用一个染料手动改变安全灯的颜色（你可以使用斧除去颜色）",
+  "advancement.minerslung:signal_torch": "噢，出——口",
+  "advancement.minerslung:signal_torch.desc": "右键一个火把使其喷出粒子，也许是用来标记洞穴的出口",
```

| **审查级别**    | 备注 |
|--------------|----|
| ⚪ 轻微 / 不阻止合并但需三思  |    |
```suggestion
  "advancement.minerslung:signal_torch.desc": "右击火把使其喷出粒子，应该是为了标记洞穴的出口",
```
原文：**Right-click a torch to make it spew particles, perhaps to signal the exit of a cave**